### ISSUE TITLE
Window content jumps down than back up after every command / flickering / stuttering

### ISSUE THREAD

**Issue by wkiefer** [warpdotdev/Warp#743]
### Describe the bug

```shell
In compact or full mode, when I type Enter after any command (e.g. `ls`) — the prompt and the line below disappear briefly, which shifts the entire content down two lines, then the command output comes and it all shifts back up.

Ideally it would not shift down and flicker. It's the primary reason I can't try warp as my main terminal right now.
```

### To Reproduce

Steps to reproduce:
1. Type `ls` 5-7 times so your terminal is full of content.
2. Type `ls` and hit enter
3. Notice all the text jumps down 1-3 lines briefly before the new output is shown

### Operating System

MacOS

### OS Version

12.2

**Comment by elviskahoro** [comment]
@wkiefer Can you send a video of this! It will help us debug!

**Comment by wkiefer** [comment]
Sure thing! Here it is:

https://user-images.githubusercontent.com/103811/153760559-b15c75a0-2517-4834-800a-014b6e3dcb58.mov

**Comment by wkiefer** [comment]
@elviskahoro Does this not repro for everyone else? it happens on both my computers 100% of the time.

**Comment by MiguelSOliveira** [comment]
i get this exact behavior as well

**Comment by elviskahoro** [comment]
From @patrick91 via #743

### Describe the bug

```shell
this might be annoying just for me, but I thought I'd report it in case other people get the same. It might also be a configuration issue, I've tried with different settings but nothing fixed it :)

TLDR: when running any command the UI jumps quite a bit and it feels frustrating to me
```


### To Reproduce

Run any command (see the video attached)

### Expected behaviour

```shell
The ui shouldn't jump this much 😊
```


### Screenshots

![2022-02-18 12 14 51](https://user-images.githubusercontent.com/667029/154739780-dc922284-211a-43ef-87e7-7f13281d452c.gif)


### Operating System

MacOS

### OS Version

12.1

### Additional context

_No response_

**Comment by elviskahoro** [comment]
This was fixed recently! Apologies for the delay and thanks for opening.
